src/formatting_layer.rs

```diff
@@ -75,6 +76,18 @@ impl<W: for<'a> MakeWriter<'a> + 'static> BunyanFormattingLayer<W> {
             pid: std::process::id(),
             hostname: gethostname::gethostname().to_string_lossy().into_owned(),
             bunyan_version: 0,
+            default_fields: Vec::new(),
```

Comment:
`new` should call `with_default_fields` internally passing an empty set of fields - we don't want to duplicate the initialisation logic for the other fields.

Reply:
Fixed